### DISCUSSION TITLE
Adicionar Links no meu Readme

### DISCUSSION THREAD

**Discussion by PsyDrops** [PsyDrops/PsyDrops#1] [Q&A] [answered]
Devo utilizar o elemento HTML `<a>` ou a sintaxe do markdown através de `[Link](Endreço)` ?

**Comment by wesleyallan** [comment, marked as answer]
Depende do que você busca, se sua ideia e abrir em uma nova aba recomendo usar o elemento HTML `<a>` com o `_blanck`, caso não se importe que seja redirecionado utilize a sintaxe do Markdown.
